Default.sublime-keymap

```diff
@@ -135,57 +135,6 @@
             { "key": "selector", "operator": "equal", "operand": "source.ts" },
             { "key": "is_popup_visible"}
         ]
-    },    // In case when auto match is enabled, only format if not within {}
-    {
-        "keys": [ "}" ],
-        "command": "typescript_format_on_key",
-        "args": { "key": "}" },
-        "context": [
-            { "key": "selector", "operator": "equal", "operand": "source.ts" },
-            { "key": "setting.typescript_auto_format", "operator": "equal", "operand": true },
-            { "key": "setting.auto_match_enabled", "operator": "equal", "operand": true },
-            { "key": "num_selections", "operator": "equal", "operand": 1},
-            { "key": "selection_empty", "operator": "equal", "operand": true },
-
-            { "key": "preceding_text", "operator": "not_regex_contains", "operand": "\\{$" },
-            { "key": "following_text", "operator": "not_regex_contains", "operand": "^\\}" }
-        ]
-    },
-    // In case when auto match is disabled, format the block regardless
-    {
-        "keys": [ "}" ],
-        "command": "typescript_format_on_key",
-        "args": { "key": "}" },
-        "context": [
-            { "key": "num_selections", "operator": "equal", "operand": 1},
-            { "key": "setting.typescript_auto_format", "operator": "equal", "operand": true },
-            { "key": "selection_empty", "operator": "equal", "operand": true, "match_all": true },
-            { "key": "selector", "operator": "equal", "operand": "source.ts" },
-            { "key": "setting.auto_match_enabled", "operator": "equal", "operand": false }
-        ]
-    },
-    {
-        "keys": [ ";" ],
-        "command": "typescript_format_on_key",
-        "args": { "key": ";" },
-        "context": [
-            { "key": "selection_empty", "operator": "equal", "operand": true, "match_all": true },
-            { "key": "setting.typescript_auto_format", "operator": "equal", "operand": true },
-            { "key": "selector", "operator": "equal", "operand": "source.ts" }
-        ]
-    },
-    {
-        "keys": [ "enter" ],
-        "command": "typescript_format_on_key",
-        "args": { "key": "\n" },
-        "context": [
-            { "key": "setting.typescript_auto_indent", "operator": "equal", "operand": true },
-            { "key": "auto_complete_visible", "operator": "equal", "operand": false },
```

Comment:
Most of the conditions are incorporated: 
- ``selection_empty == True`` -> ``info.prev_sel[0].empty()``
- ``setting.typescript_auto_indent`` -> ``cli.ts_auto_indent_enabled``
- ``num_selections == 1`` -> ``len(info.prev_sel) == 1``
- ``selector contains 'meta.scope.between-tag-pair' `` -> ``view.score_selector(pos, "meta.scope.between-tag-pair") > 0``
etc.

The only change I made was regarding the ``auto_complete_visible``, I am not sure we need that condition. To be more specific, I think we still need to do the formatting if the user types an `;` when the auto completion list is up.